outline/setup_win7.md

```diff
@@ -56,7 +56,7 @@ Go to the [Light Table site](http://www.lighttable.com/). On the page there, you
 ![Light Table downloads](img/light-table-download.png)
 ![Light Table downloads Windows](img/win/light-table-download.png)
 
-Unzip this file (either by finding it in your Downloads folder and double-clicking it, or by choosing "Open" when downloading.) Inside the .zip file, there is a a directory called "Light Table". Drag this to your desktop. (If you know what you are doing and want this somewhere else, that is fine.)
+Unzip this file (either by finding it in your Downloads folder and double-clicking it, or by choosing "Open" when downloading.) Inside the .zip file, there is a a directory called "Light Table". Drag this to your desktop. (If you know what you are doing and want this somewhere else, that is fine; however, the location you choose should not have spaces anywhere in the file path.)
```

Comment:
Sure, succinct is better. Actually, even the Desktop location could be problematic, if the user profile has a space in it (i.e. c:\users\my whole name\desktop). What about:

_". . . Drag this to your desktop, or another location of your choosing. Be sure the location you choose does not have spaces anywhere in the file path._